learn/experimental/vector_search.mdx

```diff
@@ -58,10 +58,20 @@ curl \
   }'
 ```
 
-`documentTemplate` is an optional field you can use to send only relevant data to the model. `documentTemplate` must be a [Liquid template](https://shopify.github.io/liquid/).
+`apiKey` is a mandatory field when using an OpenAI embedder. [Generate an API key](https://platform.openai.com/api-keys) from your OpenAI account. Use [tier 2 keys](https://platform.openai.com/docs/guides/rate-limits/usage-tiers?context=tier-two) for optimal performance.
```

Comment:
Thank you for adding the requested information. A detail, `apiKey` is not mandatory in that we take the key from the environment if it wasn't specified. I think it is important to specify because it could be surprising to users that we use this env variable.

```suggestion
It is mandatory to pass an OpenAI API key through the `OPENAI_API_KEY` environment variable or the `apiKey` field when using an OpenAI embedder. [Generate an API key](https://platform.openai.com/api-keys) from your OpenAI account. Use [tier 2 keys](https://platform.openai.com/docs/guides/rate-limits/usage-tiers?context=tier-two) or above for optimal performance.
```